### ISSUE TITLE
ng add @ngx-builders/analyze fails in Angular 13

### ISSUE THREAD

**Issue by brodziakm** [ngx-builders/source-map-analyzer#83]
When running the command:

`ng add @ngx-builders/analyze`

the following failure is reported:

`An unhandled exception occurred: NOT SUPPORTED: keyword "id", use "$id" for schema ID`

with the following detail:

```
[error] Error: NOT SUPPORTED: keyword "id", use "$id" for schema ID
    at Object.code (.\node_modules\@angular-devkit\core\node_modules\ajv\dist\vocabularies\core\id.js:6:15)
    at keywordCode (.\node_modules\@angular-devkit\core\node_modules\ajv\dist\compile\validate\index.js:454:13)
    at .\node_modules\@angular-devkit\core\node_modules\ajv\dist\compile\validate\index.js:222:17
    at CodeGen.code (.\node_modules\@angular-devkit\core\node_modules\ajv\dist\compile\codegen\index.js:439:13)
    at CodeGen.block (.\node_modules\@angular-devkit\core\node_modules\ajv\dist\compile\codegen\index.js:568:18)
    at iterateKeywords (.\node_modules\@angular-devkit\core\node_modules\ajv\dist\compile\validate\index.js:219:9)
    at groupKeywords (.\node_modules\@angular-devkit\core\node_modules\ajv\dist\compile\validate\index.js:208:13)
    at .\node_modules\@angular-devkit\core\node_modules\ajv\dist\compile\validate\index.js:192:13
    at CodeGen.code (.\node_modules\@angular-devkit\core\node_modules\ajv\dist\compile\codegen\index.js:439:13)
    at CodeGen.block (.\node_modules\@angular-devkit\core\node_modules\ajv\dist\compile\codegen\index.js:568:18)

```

**Comment by patelvimal** [comment]
@santoshyadavdev id in schema was deprecated and  I guess now removed in v13.
reference:- 
https://github.com/angular/angular-cli/releases/tag/13.0.0

We might need to update @angular-devkit/core version and other stuffs, I will take this up. Will raise the PR once ready.

**Comment by patelvimal** [comment]
@brodziakm , @santoshyadavdev This issue has been fixed under version 3.0.1 We can mark this as close.

**Comment by brodziakm** [comment]
Thanks for sorting this out.
